src/helix-ui/elements/HXFileInputElement.less

```diff
@@ -1,59 +1,25 @@
 @import './HXElement';
-@import (reference) "components/buttons";
 
-:host {
-  #hxContent {
-    /*
-      Needed to prevent focusing of native file inputs.
-      Especially true for Microsoft browsers, because
-      Edge and IE have double tab stops.
-    */
-    display: none;
-  }
-
-  #hxButton {
-    #Button.reset();
-    #Button.basic();
-    #Button.secondary();
-    #Button.medium();
-    width: 100%;
-
-    #hxLabel {
-      margin: 0;
-    }
-  }
+#hxFileInput {
+  width: 100%;
```

Comment:
Do we need this style? `<div>`'s are typically full width, by default.

Reply:
looks like not needed to removed